docs/configuration/optional-settings.md

```diff
@@ -135,6 +135,22 @@ Enforcement of unique IP space can be toggled on a per-VRF basis. To enforce uni
 
 ---
 
+## GITHUB_REPOSITORY
```

Comment:
We'll also need a mechanism to specify the remote server, for organizations which maintain a fork on a local GitHub Enterprise server. (I'm assuming the same API is available.)

Reply:
Whether it makes sense to introduce a separate config parameter or just change this one to convey the full URL is up for discussion; I vote for the later.